package.json

```diff
@@ -20,13 +22,17 @@
     "babel-plugin-transform-react-jsx": "^6.22.0",
     "babel-preset-es2015": "^6.22.0",
     "babili-webpack-plugin": "0.0.10",
+    "browserify": "^14.0.0",
     "css-loader": "^0.26.1",
     "eslint": "^3.14.1",
     "eslint-config-airbnb-base": "^11.0.1",
     "eslint-plugin-import": "^2.2.0",
     "extract-text-webpack-plugin": "^2.0.0-rc.3",
+    "hyperxify": "^2.0.1",
```

Comment:
Yea I can change that but even then check this: https://github.com/selfup/hyperapp-one/issues/4#issuecomment-278679924